src/domain/Car.js

```diff
@@ -0,0 +1,24 @@
+class Car {
+  #name;
+
+  #location;
```

Comment:
사소하지만, 개행은 없으면 어떨까요? 
코드 상에서 `개행` 또한 코드 작성자의 `의도`를 표현하는데요, 멤버변수끼리 구분할 특별한 의도가 없으시다면 붙여서 표현하는게 좋겠습니다.

```suggestion
  #name;
  #location;
```

Reply:
아! 관련해서 처음엔 개행 없이 작성했었는데 eslint 에서 사이에 공백을 작성하라고 하더라구요 

> Expected blank line between class members

만약 잘 사용되지 않는 설정이거나 특별한 의도가 없는 것들은 붙여서 작성하는 것이 자주 사용되는 방법일까요?

Reply:
멤버변수끼리 개행은 조금 낯서네요...! [공식문서](https://eslint.org/docs/latest/rules/lines-between-class-members) 예제 보면 커스텀해서 사용할 수 있어보이는데 그렇게 해도 좋을 것 같아요. 혹은 페어끼리의 컨벤션이라면 그대로 가져가셔도 좋고요.

Reply:
넵!! 반영해서 커스텀한 새 eslint 규칙을 적용하는 방안으로 해보겠습니다!